gpio.py

```diff
@@ -17,9 +17,10 @@ class PinState(object):
         value: the file pointer to set/read value of pin.
         direction: the file pointer to set/read direction of the pin.
     """
-    def __init__(self, value, direction):
+    def __init__(self, value, direction, active_low):

```

Comment:
should have a default. `active_low = None`. If it is `None` it should not pass it to the `PinState`

Reply:
Due to the wav `_verify()` woks, it's never possible for `active_low` to be None. Not sure exactly what you mean by not passing to `PinState` if it's `None`, as it's already passed here. I changed `setup()` to allow `active_low` to be `None` at that point which I think address what you are getting at. Let me know if that is not true.